linkerd/app/outbound/src/opaq/logical.rs

```diff
@@ -84,31 +112,64 @@ impl<N> Outbound<N> {
                 // Lazily cache a service for each `RouteParams`
                 // returned from the `SelectRoute` impl.
                 .push_on_service(route)
-                .push(svc::NewOneshotRoute::<Params, _, _>::layer_cached());
+                .push(svc::NewOneshotRoute::<Params<T>, _, _>::layer_cached());
 
             // For each `Logical` target, watch its `Profile`, maintaining a
             // cache of all concrete services used by the router.
             concrete
```

Comment:
this comment should maybe be updated since we're not doing this for each `Logical` target, but only for the `Routable` ones, and the stack we build here now contains a switch?